static/js/tools/stat_var/info.tsx

```diff
@@ -24,12 +24,13 @@ class Info extends Component {
   render(): JSX.Element {
     return (
       <div id="placeholder-container">
-        <h1 className="mb-4">Statistical Variable Explorer</h1>
         <p>
```

Comment:
Why is the h1 title removed?

Reply:
this was moved to the parent component (page.tsx) because now we always show this header regardless of whether a stat var is selected or not